authors/apps/authentication/views.py

```diff
@@ -135,3 +140,63 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ForgetPasswordAPIView(APIView):
+    """Forget password view captures email and generates token that will be.
+    used during reset password. Data that is captures in the view is send to
+    the serializer class
+    """
+    permission_classes = (AllowAny,)
+    serializer_class = ForgetPasswordSerializer
+
+    def post(self, request):
+        """Captures data entered by user and generates token"""
+
+        email_object = request.data
+        user_email = email_object['email']
```

Comment:
We can use a single line here by grabbing user_email from request.data directly

Reply:
I have managed to grab the email directly and also the data entered by user.